qiskit/quantum_info/operators/random.py

```diff
@@ -64,18 +64,18 @@ def random_hermitian(dims, traceless=False, seed=None):
         traceless (bool): Optional. If True subtract diagonal entries to
                           return a traceless hermitian operator
                           (Default: False).
-        seed (int or RandomState): Optional. Set a fixed seed or
+        seed (int or default_rng): Optional. Set a fixed seed or
```

Comment:
Should this be
```suggestion
        seed (generator_seed or Generator): ...
```
(and same for all the other locations)
Looking at the documentation for the new `Generator` it accepts a lot of different things for seed values:
```
seed : {None, int, array_like[ints], ISeedSequence, BitGenerator, Generator}, optional
        A seed to initialize the `BitGenerator`. 
```

Reply:
But we do not need to use all of them.  That seems outside of this PR.

Reply:
i agree this is future pr